test/unit/test_keyring_multi.py

```diff
@@ -51,28 +45,44 @@
 _WRAPPING_KEY_AES = b"\xeby-\x80A6\x15rA8\x83#,\xe4\xab\xac`\xaf\x99Z\xc1\xce\xdb\xb6\x0f\xb7\x805\xb2\x14J3"
 _SIGNING_KEY = b"aws-crypto-public-key"
 
-mock_generator = MagicMock()
-mock_generator.__class__ = RawAESKeyring
 
-mock_child_1 = MagicMock()
-mock_child_1.__class__ = RawAESKeyring
+@pytest.fixture
+def identity_keyring():
+    return IdentityKeyring()
+
+
+@pytest.fixture
+def keyring_which_only_generates():
+    return OnlyGenerateKeyring()
+
+
+@pytest.fixture
+def mock_generator():
+    mock_generator = MagicMock()
```

Comment:
As a general good practice, we should avoid overriding a name from a higher scope (in this case, the function name).